app/code/Magento/Ui/view/base/web/templates/form/field.html

```diff
@@ -40,6 +40,18 @@
 
         <div class="admin__additional-info" if="$data.additionalInfo" html="$data.additionalInfo"></div>
 
+        <div class="admin__scope-hints" if="$data.scopeHints">
```

Comment:
Please, create the new custom template especially for the product form, it's not a good idea to define `scopeHints ` for all form fields

Reply:
@sidolov 
It won't be shown when there are no scopeHints and I think that there has to be changed to much to only use a separate template but correct me if I am wrong. Besides that it is also easy extendable for the Category Edit

Reply:
@lewisvoncken the problem is that your changes are introduced to `app/code/Magento/Ui/view/base/web/templates/form/field.html` that is used for all fields, not only product, but the PHP implementation is related only to catalog product. Please create a template that will be used only for catalog product form.